client/my-sites/checkout/utils.ts

```diff
@@ -135,3 +137,22 @@ export function getAffiliateCouponLabel(): string {
 	// translators: The label of the coupon line item in checkout
 	return translate( 'Exclusive Offer Applied' );
 }
+
+export function getWpcomPlanTotalIfPaidMonthly(
+	product: ResponseCartProduct,
+	plans: any
+): number | undefined {
+	if ( isWpComPlan( product?.product_slug ) && ! isMonthly( product?.product_slug ) ) {
+		const monthlyPlanSlug = getMonthlyPlanByYearly( product?.product_slug );
+		if (
+			monthlyPlanSlug &&
+			plans?.[ monthlyPlanSlug ] &&
+			product.months_per_bill_period &&
+			plans[ monthlyPlanSlug ].pricing?.originalPrice?.monthly
```

Comment:
We should really be using `usePricingMetaForGridPlans` for any pricing related info. Unless there is a reason not to, avoid using the Query hooks for pricing.

Reply:
Done.